addons/point_of_sale/static/tests/tours/product_screen_tour.js

```diff
@@ -328,3 +328,49 @@ registry.category("web_tour.tours").add("PosCustomerAllFieldsDisplayed", {
             PartnerList.searchCustomerValue("john@doe.com"),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("PosPopupPriceAndQuantity", {
+    test: true,
+    url: "/pos/ui",
+    steps: () =>
+        [
+            Dialog.confirm("Open session"),
+            ProductScreen.clickInfoProduct("Test Product with Sizes"),
+            {
+                trigger: ".section-product-info-title-quantity",
+                run: () => {
+                    const spanQuantity = document.querySelector(
```

Comment:
If possible avoid doing checks in the `run` and just make another trigger. In this case something like
`trigger: ".section-product-info-title-quantity:contains("20 Units")"`